vbuild/apkbuild.py

```
@@ -26,6 +34,62 @@ def wrapper(self: "APKBUILD") -> str | None:
+
+def quoted_string(value:str) -> str:
+    offset = 0
+    quoted_value=""
+    while True:
+        if offset >= len(value):
+            break
+
+        token = value[offset]
+        offset += 1
+
+        if token != "$":
```

Ruff (0.15.1)

[error] Possible hardcoded password assigned to: "token"

(S105)